packages/gestalt/src/ButtonGroup.tsx

```diff
@@ -16,15 +17,26 @@ type Props = {
  *
  */
 function ButtonGroup({ children }: Props) {
+  const isInVRExperiment = useInExperiment({
+    webExperimentName: 'web_gestalt_visualRefresh',
+    mwebExperimentName: 'web_gestalt_visualRefresh',
+  });
+
   if (Children.count(children) === 0) {
     return null;
   }
 
   return (
     <Box display="flex" margin={-1} wrap>
-      {Children.map(children, (child) =>
-        child !== null && child !== undefined ? <Box padding={1}>{child}</Box> : null,
-      )}
+      {isInVRExperiment
+        ? Children.map(children, (child) =>
+            child !== null && child !== undefined ? (
+              <div style={{ padding: 2 }}>{child}</div>
```

Comment:
this is problematic because they don't even have tokens for 2 px!

Reply:
Using Flex instead